interface.py

```diff
@@ -92,11 +97,13 @@ def switch_power_down(self):
                 self.power_down = True
             else:
                 self.power_down = False
+        self.change_callback()
 
     def confirm_selection(self):
         """
         When indicator is False the player can choose cards and switch Power Down.
         When is True the player ended the selection of cards.
         """
         if None not in self.program:
-            self.selection_confirmed = True
+            self.confirm_selection = True
```

Comment:
Menis zde nazev atributu, ale je to jedine misto, kde se meni. Proto hra pada.

Reply:
jo tady je problém :-D děkuji.